docs/contents/components/forms/year-picker/index.mdx

```diff
@@ -28,59 +28,95 @@ import { YearPicker } from "@yamada-ui/calendar"
 
```

Comment:
@kaehehehe 
This is the same as `docs/contents/components/forms/year-picker/index.ja.mdx`.